packages/components/src/components/post-popovercontainer/post-popovercontainer.tsx

```diff
@@ -47,6 +47,7 @@ export class PostPopovercontainer {
   private arrowRef: HTMLElement;
   private eventTarget: Element;
   private clearAutoUpdate: () => void;
+  private toggleTimeout: number;
```

Comment:
At first read, I thought that this variable contain an actual time value, but it's rather the id of the setTimeout. Also it can contain null on line 131